open-api/immich-openapi-specs.json

```diff
@@ -11775,6 +11777,9 @@
           "extractEmbedded": {
             "type": "boolean"
           },
+          "fullsizePreview": {
```

Comment:
I was thinking of something more like `"fullsize": {"enabled": false, "format": "jpeg"}`, where `format` is ignored if using the extracted preview.

Reply:
In the past, we've gotten complaints about image settings being shared between different files, and there may be other options related to the fullsize image added down the line. For example, a quality option might be added if users are concerned about file size and want 90 or 95 quality instead of 100, Safari users might complain because they don't need fullsize images for HEIF, etc.

I think the config field should be an object to make it easier to add settings in the future. The format is maybe superfluous in a vacuum, but the fact that one setting (format) is shared between the preview and the fullsize and the other two (quality, resolution) are not can lead to confusion. Having them completely separate will be less ambiguous.

Reply:
it is now a separate `config.image.fullsize` config object.